### PR TITLE
task3.5 completed

### PR DESCRIPTION
**Pull request by joynal-miah**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -12,6 +12,17 @@ afterAll(() => {
   return db.end();
 });
 
+describe("GET /api", () => {
+  it("should return the endpoints object", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        expect(response.statusCode).toBe(200);
+        expect(response.body).toEqual(expect.any(Object));
```
Comment (Augs0) [inline __tests__/app.test.js:22]: We don't want this to be any object - we want to make sure it's the object we're looking for. You have two options here:
- bring in the endpoints file and test it's equal to your response
- iterate over the response and check the keys are right i.e. slug and description

**Review by Augs0 — CHANGES_REQUESTED** [review]
Good work but please update the test according to the feedback

File: __tests__/app.test.js [hunk 2]
```diff
@@ -17,8 +18,11 @@ describe("GET /api", () => {
     return request(app)
       .get("/api")
       .then((response) => {
+        console.log(response.body)
```
Comment (pippaamy) [inline __tests__/app.test.js]: we don't want to keep console.logs up, delete these before pushing

**Review by pippaamy — COMMENTED** [review]
great job, lets avoid keeping console logs in!

**Pull request merged**